src/components/Documentation.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+    <section class="section" id="dochome">
+    <div class="columns is-mobile is-centered">
+      <div class="column is-three-quarters-tablet is-half-desktop has-text-centered">
+        <div class="block">
+          <h1 class="title is-1">Documentation</h1>
+          <p>Resources for using Pangeo Forge</p>
+        </div>
+      </div>
+    </div>
+    <div class="content" id="firststeps">
+        <h3 class="title is-3">First steps</h3>
+        <p>New to Pangeo Forge?  Start here!</p>
+        <ul>
+            <li><strong>What is Pangeo Forge?</strong> - Read more about what Pangeo Forge is and
+            how it works.</li>
+            <li><strong>Introduction Tutorial</strong> - Ready to code?  Walk through creating and
+            deploying your first data transformation pipeline.</li>
+        </ul>
+        <hr class="solid">
+    </div>
+    <div class="content" id="organization">
+        <h3 class="title is-3">How the documentation is organized</h3>
+        <p>There are a number of places to access resources when working with components of Pangeo
+        Forge.  Here is an overview of what you will find:</p>
+        <ul>
+            <li>The <strong>Introduction Tutorial</strong> is the place to start with Pangeo Forge.
+            It walks the user through the process of getting setup with their first data transformation.</li>
+            <li><strong>Guides</strong> explain core Pangeo Forge concepts in detail. They provide
```

Comment:
Do you have a link of a Guide for another open source package that you really like? I'm having difficulty picturing what the Guide looks like.

Reply:
I think the [existing documentation](https://pangeo-forge.readthedocs.io/en/latest) for `pangeo_forge_recipes` can evolve into a user guide. The [Xarray User Guide](http://xarray.pydata.org/en/stable/user-guide/index.html) is a good inspiration.

Reply:
I like the xarray User Guide pretty well.  When I'm saying _Guides_ I'm thinking of the "HowTo" (goal oriented: "take the reader through the steps required to solve a real-world problem") and "Explanation" (understanding oriented: "clarify and illuminate a particular topic. They broaden the documentation’s coverage of a topic.") sections of the [Divio](https://documentation.divio.com/) documentation construct.  
While that was my original thought I think we can think more fluidly than that if we want.  xarray sort of does them together and I think those docs are nice.